app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
+    return false if obstructed_diagonally?(to_x:, to_y:)
+    return false if obstructed_vertically?(to_x:, to_y:)
+    return false if obstructed_horizontally?(to_x:, to_y:)
+    current_color = current_piece.color
```

Comment:
Right, color and other attributes are implicit in each piece object, so no need for that line #101. So then, does @asmentko actually need to develop the `current_piece` helper method (as shown in Trello's In Progress column)?